### DISCUSSION TITLE
Does Rare support achievements?

### DISCUSSION THREAD

**Discussion by bialyikar** [RareDevs/Rare#303] [Q&A] [answered]
Does Rare support achievements from the Epic Store? As far as I know Legendary doesn't support them.

**Comment by loathingKernel** [comment, marked as answer]
No, it doesn't. We have discussed ways to show achievements on the client but nothing has been implemented yet.

I don't know how registering achievements is handled in the Epic client but that is unlikely something we are going to implement if it is provided by the client and not the game itself.
